scenario.scala

```diff
@@ -38,14 +38,22 @@ object Walk {
     .get("/electronics/")
     .headers(headers_classic)
 
-  val categoryListFilterPrice = http("Category list filtering")
+  val categoryListFilterPrice = http("Category list filtering by price")
     .get("/electronics/?product_filter_form[minimalPrice]=0&product_filter_form[maximalPrice]=20000")
     .headers(headers_ajax)
 
-  val categoryListFilterPriceBrand = http("Category list filtering")
+  val categoryListFilterPriceBrand = http("Category list filtering by price with brand selected")
     .get("/electronics/?product_filter_form[minimalPrice]=0&product_filter_form[maximalPrice]=20000&product_filter_form[brands][]=4")
     .headers(headers_ajax)
 
+  val categoryListFilterParameters = http("Category list filtering by parameters")
+    .get("/electronics/?=product_filter_form[parameters][3][]=6&product_filter_form[parameters][4][]=8&product_filter_form[parameters][4][]=12")
+    .headers(headers_ajax)
+
+  val categoryListFilterAllFilters = http("Category list filtering by all possible filters")
+    .get("/electronics/?product_filter_form[minimalPrice]=200&product_filter_form[maximalPrice]=30000&product_filter_form[inStock]=1&product_filter_form[brands][]=5&product_filter_form[parameters][5][]=8&product_filter_form[parameters][1][]=2&product_filter_form[parameters][1][]=14&product_filter_form[parameters][4][]=8")
```

Comment:
can you pick smaller range for prices? 200 and 30000 are now in last demo data build over the real MIN and MAX from electronics.

Reply:
actually there is available also options in parameters filter
Resolution (id 3 value 6)
Technology  (id 2 value 4)
and Flags filter is not used

Reply:
OK

Reply:
RESOLVE